resolving/npm-resolver/test/index.ts

```diff
@@ -273,8 +274,10 @@ test('can resolve aliased dependency w/o version specifier to default tag', asyn
   })
   const resolveResult = await resolveFromNpm({ alias: 'positive', pref: 'npm:is-positive' }, {
     defaultTag: 'stable',
+    calcSpecifier: true,
   })
   expect(resolveResult!.id).toBe('is-positive@3.0.0')
+  expect(resolveResult!.specifier).toBe('npm:is-positive@^3.0.0')
```

Comment:
I prefer this because when the test fails, it will let us know whether just the `id` fails or both `id` and `specifier`:

```suggestion
  expect(resolveResult).toMatchObject({
    id: 'is-positive@3.0.0',
    specifier: 'npm:is-positive@^3.0.0',
  })
```

Reply:
Actually, there are more assertions like this. Nevermind.